go-controller/pkg/ovn/ovn.go

```diff
@@ -64,6 +61,10 @@ type namespaceInfo struct {
 	hybridOverlayVTEP       net.IP
 
 	multicastEnabled bool
+
+	// addressSet is the name of an address set that holds the IP addresses
+	// of all pods in the namespace.
+	addressSet AddressSet
```

Comment:
it's not just "the name" of the address set. (Also, any reason you moved this from the top of the struct to the bottom?)

Reply:
nope, moved back up